.github/workflows/ci.yml

```diff
@@ -31,11 +25,6 @@ jobs:
         run: yarn test
       - name: Build
         run: yarn build:local
-      - name: SonarCloud Scan
-        uses: sonarsource/sonarcloud-github-action@master
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

Comment:
let's use the latest version of `artiomtr/jest-coverage-report-action` and configure for annotations